Assets/Scripts/Inventory/Inventory_System.cs

```diff
@@ -19,9 +19,7 @@ public class Inventory_System : MonoBehaviour {
     private int SeedNumber;
```

Comment:
Jest ok, aktualizuj tylko UI posiadanych nasionek - kiedy się skończą to powinny zniknąć z EQ